src/utils/results-reporter.ts

```diff
@@ -85,33 +87,56 @@ export class ResultsReporter {
     const screensWithDifferences = screenshotDiffResults.filter(
       (result) => result.outcome === "diff"
     ).length;
-    const totalScreens = screenshotDiffResults.length;
+    const totalScreensCompared = screenshotDiffResults.length;
+    const totalScreenshotsTaken = testCaseResults.reduce(
+      (total, testCase) => total + testCase.totalNumberOfScreenshots,
+      0
+    );
 
     if (screensWithDifferences === 0) {
       if (!(testRun.project as EnrichedProject).isGitHubIntegrationActive) {
         await this.setCommitStatus({
-          description: `Zero differences across ${totalScreens} screens tested`,
+          description: `Zero differences across ${totalScreensCompared} screens tested`,
           state: "success",
           targetUrl: testRun.url,
         });
       }
-      if (totalScreens > 0) {
+      if (totalScreensCompared > 0) {
         await this.setStatusComment({
           createIfDoesNotExist: true,
-          body: `✅ Meticulous spotted zero visual differences across ${totalScreens} screens tested: [view results](${testRun.url}).`,
+          body: `✅ Meticulous spotted zero visual differences across ${totalScreensCompared} screens tested: [view results](${testRun.url}).`,
         });
+      } else {
+        if (totalScreenshotsTaken === 0) {
+          await this.setStatusComment({
+            createIfDoesNotExist: true,
+            body: `❌ Meticulous replayed ${testCaseResults.length} user sessions, but no screenshots were taken. This likely means there was an error replaying the sessions. Please view the logs of the Github workflow.`,
```

Comment:
What about the case when they've recorded no sessions yet? Feel like this message could be misleading in that case?

Reply:
In this case the whole action run will fail. Errors will be reported in (https://github.com/alwaysmeticulous/report-diffs-action/blob/7004c5c983313cbc1c629a410281eb80d78c849f/src/action.ts#L193)